CMakeLists.txt

```diff
@@ -100,6 +100,11 @@ IF (DEPRECATE_HARD)
 	ADD_DEFINITIONS(-DGIT_DEPRECATE_HARD)
 ENDIF()
 
+# Installation paths
```

Comment:
Does it matter if you move it higher up ? I kinda like that the whole option setup is around the top of the file, as a documentation source. Maybe gathering all *known usable knobs* would ease some issues ? That's completely cosmetic though, but given that CMake code has a tendency to ~melt~ make you doubt from where it's coming from. Here, I was kinda hoping those variables were global CMake variables we're defining for clarity, now I don't know 😉.